Header header
string robot_id
bool is_ready
bool is_arrived
float32 x_initial
float32 y_initial
